source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -561,8 +561,8 @@ WaitQueueTimeout, if the driver supports one and it was specified by the user.
 
 If the pool is "closed" or "paused", any attempt to check out a `Connection
 <#connection>`_ MUST throw an Error. The error thrown as a result of the pool
-being "paused" MUST be considered a non-timeout network error for the purposes
-of retryability and monitoring.
+being "paused" MUST be considered a retryable error and MUST NOT be an error
+that marks the SDAM state unknown.
```

Comment:
@patrickfreed do you have any ideas for how to test this change? What are your thoughts on my alternative idea where pool.ready() increments the generation number when the state changes from "paused" to "ready"?

Reply:
Great points all around. I changed this section to a MUST because I think we should standardize error handling in SDAM. I haven't changed the generation behavior since I don't think it's necessary in this PR.

Reply:
> do the Java or C# drivers update the topology based on errors in the background thread while ensuring minPoolSize?

No for Java.  Probably not for C# but haven't confirmed.

bq. would it be possible for them to do so?

I don't see why not.  Currently, the connection pool implementations have no reference to the topology.  The reference is the other way round.  But we can introduce some sort of invalidation callback.

Reply:
Awesome, then it seems like we're safe to upgrade this SHOULD -> MUST then.